nx-dev/ui-common/src/lib/youtube.component.tsx

```diff
@@ -124,7 +124,7 @@ export function YouTube(props: YouTubeProps): JSX.Element {
               {
                 'rounded-lg shadow-lg': !props.disableRoundedCorners,
               },
-              'border-2 border-slate-200 hover:border-slate-500 dark:border-slate-700/40 dark:hover:border-slate-700'
+              'aspect-video border-2 border-slate-200 hover:border-slate-500 dark:border-slate-700/40 dark:hover:border-slate-700'
```

Comment:
Does this have  any effect on the existing Next.js docs rendering?

Reply:
no this was just fixing the styles I noticed where the youtube video didn't have the right ratio. it seems that the previous impl was done via a global css instead of tailwinds which is why it didn't have the right aspect ratio initially when porting.